src/EventListener/FormListener.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Bolt\EventListener;
```

Comment:
Maybe sub-namespaces then. I just see it getting cluttered very fast. What about `Bolt\EventListener\Kernel` and `Bolt\EventListener\Form`? Hmm...but that's the same as having `Bolt\Providers` which I don't like either. I would rather the logic be grouped together by topic.